app/models/enterprise.rb

```diff
@@ -317,6 +317,14 @@ def white_label_logo_url(name = :default)
     )
   end
 
+  def white_label_logo_link
+    return nil if self[:white_label_logo_link].blank?  
+
+    return self[:white_label_logo_link] if self[:white_label_logo_link].start_with?('http')
+
+    "http://#{self[:white_label_logo_link]}"
```

Comment:
Shouldn't we use https by default? :shrug: 

Or maybe we should validate the URL on the model earlier. :thinking: